app/models/webhook_header.rb

```diff
@@ -0,0 +1,22 @@
+# == Schema Information
+#
+# Table name: webhook_headers
+#
+#  id         :integer          not null, primary key
+#  webhook_id :integer
+#  name       :string(255)
+#  value      :string(255)
```

Comment:
>>    Why do you want to filter out delete events?
>
> @flavio use case for delete notifications would be synchronisation of repos between different registries.

I never said I want to filter delete events. On the opposite I said in the future the will support them (they have just been introduced, I don't know if we have the code to handle them yet).